changelog/8.6.0.yaml

```diff
@@ -0,0 +1,283 @@
+      timestamp: 1667571017
+      file:
+        name: 1667571017-Add-support-for-running-the-elastic-agent-shipper.yaml
+        checksum: b2f5a0a8f9933c9ee624457116295aa674ef2a23
+    - kind: feature
+      summary: Add Common Expression Language input mapping to the Filebeat spec.
+      description: ""
+      component: spec
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1719
+      issue: []
+      timestamp: 1668483030
+      file:
+        name: 1668483030-common-expression-language.yaml
+        checksum: 82427fc514372c067e140837177c8d81131ed424
+    - kind: feature
+      summary: Reimplement processes route
+      description: Re-implement processes route to restore compatibility with the current cloud health checks
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1773
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1731
+      timestamp: 1669159455
+      file:
+        name: 1669159455-reimplement-processes-route.yaml
+        checksum: cbeb5b5766bf733dd5e200a69353e326c642a971
+    - kind: feature
+      summary: Capture stdout/stderr of all spawned components and adjust default log level to info for all components
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1702
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/221
+      timestamp: 1669236059
+      file:
+        name: 1669236059-Capture-stdout-stderr-of-all-spawned-components-to-simplify-logging.yaml
+        checksum: b4cf9cff0ef13bd168cadc80f89c4969b49fb378
+    - kind: bug-fix
+      summary: Fix Elastic-Agent fails to re-start during upgrade
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1805
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1788
+      timestamp: 1669665017
+      file:
+        name: 1669665017-fix-nil-action-ack-crash-after-local-upgrade.yaml
+        checksum: 80fb772957944cd3687aa999595d6be1b4d96d2a
+    - kind: bug-fix
+      summary: Fix osqueryd and osquery-extension.ext permissions
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1829
+      issue: []
+      timestamp: 1669739947
+      file:
+        name: 1669739947-fix-osqueryd-and-osquery-extension.ext-permissions.yaml
+        checksum: cc8d9c7ab91b7e4c45a11b5b1347238d3e757453
+    - kind: feature
+      summary: Add agent logging level to fleet configuration
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1856
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1853
+      timestamp: 1669845868
+      file:
+        name: 1669845868-add-agent-logging-level-to-fleet-configuration.yaml
+        checksum: 33cbf01b52f8cf30971d84fadbba10163197fe61
```

Comment:
```suggestion
```

This only affects communication between the endpoint and agent process, not user visible.